src/utils/logger.ts

```diff
@@ -78,13 +77,13 @@ export function wrapper(func: Function) :
       log(Severity.INFO, 'request', func.name, traceID, reqInfo);
 
       const res = await func(call, callback);
-      apm.endTransaction(statusToString(grpc.status.OK));
+      apm.endTransaction(grpc.status[grpc.status.OK]);
       const resInfo: object = extractResLog(res);
       log(Severity.INFO, 'response', func.name, traceID, resInfo);
       callback(null, res);
     } catch (err) {
       const validatedErr : ApplicationError = validateGrpcError(err);
-      log(Severity.ERROR, func.name, err.message, getCurrTraceId());
```

Comment:
if you send `any` type to string, at least stringify it so it won't put [object objecy]

Reply:
done